examples/jupyter/tutorials/tutorial7-python-operator.ipynb

```diff
@@ -0,0 +1,230 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* In the following example, we create and apply a CeedOperator for the mass matrix in 1D."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.\n",
+      " 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.\n",
+      " 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]\n"
+     ]
+    }
+   ],
+   "source": [
+    "import libceed\n",
+    "import numpy as np\n",
+    "\n",
+    "ceed = libceed.Ceed()\n",
+    "\n",
+    "nelem = 15\n",
+    "p = 5\n",
+    "q = 8\n",
+    "nx = nelem + 1\n",
+    "nu = nelem*(p-1) + 1\n",
+    "\n",
+    "# Vectors\n",
+    "x = ceed.Vector(nx)\n",
+    "x_array = np.zeros(nx)\n",
+    "for i in range(nx):\n",
+    "  x_array[i] = i / (nx - 1.0)\n",
+    "x.set_array(x_array, cmode=libceed.USE_POINTER)\n",
+    "\n",
+    "qdata = ceed.Vector(nelem*q)\n",
+    "u = ceed.Vector(nu)\n",
+    "v = ceed.Vector(nu)\n",
+    "\n",
+    "# Restrictions\n",
+    "indx = np.zeros(nx*2, dtype=\"int32\")\n",
+    "for i in range(nx):\n",
+    "  indx[2*i+0] = i\n",
+    "  indx[2*i+1] = i+1\n",
+    "rx = ceed.ElemRestriction(nelem, 2, nx, 1, indx, cmode=libceed.USE_POINTER)\n",
+    "\n",
+    "indu = np.zeros(nelem*p, dtype=\"int32\")\n",
+    "for i in range(nelem):\n",
+    "  for j in range(p):\n",
+    "    indu[p*i+j] = i*(p-1) + j\n",
+    "ru = ceed.ElemRestriction(nelem, p, nu, 1, indu, cmode=libceed.USE_POINTER)\n",
+    "strides = np.array([1, q, q], dtype=\"int32\")\n",
+    "rui = ceed.StridedElemRestriction(nelem, q, q*nelem, 1, strides)\n",
+    "\n",
+    "# Bases\n",
+    "bx = ceed.BasisTensorH1Lagrange(1, 1, 2, q, libceed.GAUSS)\n",
+    "bu = ceed.BasisTensorH1Lagrange(1, 1, p, q, libceed.GAUSS)\n",
+    "\n",
+    "# QFunctions\n",
+    "qf_setup = ceed.QFunctionByName(\"Mass1DBuild\")\n",
+    "qf_mass = ceed.QFunctionByName(\"MassApply\")\n",
+    "\n",
+    "# Operators\n",
+    "op_setup = ceed.Operator(qf_setup)\n",
+    "op_setup.set_field(\"dx\", rx, bx, libceed.VECTOR_ACTIVE)\n",
+    "op_setup.set_field(\"weights\", libceed.ELEMRESTRICTION_NONE, bx,\n",
+    "                   libceed.VECTOR_NONE)\n",
+    "op_setup.set_field(\"qdata\", rui, libceed.BASIS_COLLOCATED,\n",
+    "                   libceed.VECTOR_ACTIVE)\n",
+    "\n",
+    "op_mass = ceed.Operator(qf_mass)\n",
+    "op_mass.set_field(\"u\", ru, bu, libceed.VECTOR_ACTIVE)\n",
+    "op_mass.set_field(\"qdata\", rui, libceed.BASIS_COLLOCATED, qdata)\n",
+    "op_mass.set_field(\"v\", ru, bu, libceed.VECTOR_ACTIVE)\n",
+    "\n",
```

Comment:
I think printing these operators, or at least one of them, would help the users see what is going on